soc/microchip/pic32cxsg/common/bossa.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2020 Google LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
this file already exists, move the existing file to a common place and include it

Reply:
what could be a common place for 2 different vendors?